testsuite/testsuite-jaxrs/src/test/java/org/wildfly/swarm/jaxrs/JAXRSArquillianTest.java

```diff
@@ -35,6 +36,7 @@
  * @author Bob McWhirter
  */
 @RunWith(Arquillian.class)
+@Ignore
```

Comment:
Why are this and other tests ignored?

Reply:
Looks like a merge error as this should not have been changed.